improver_tests/nbhood/recursive_filter/test_RecursiveFilter.py

```diff
@@ -278,77 +290,122 @@ def test_with_separate_mask_cube_and_nan(self):
         self.assertArrayEqual(result_nan_array, expected_nans)
 
 
-class Test__validate_smoothing_coefficients(Test_RecursiveFilter):
+class Test__validate_and_pad_coefficients(Test_RecursiveFilter):
 
-    """Test the _validate_smoothing_coefficients method"""
+    """Test the _validate_and_pad_coefficients method"""
 
     def test_smoothing_coefficients_cube(self):
         """Test that correctly shaped smoothing_coefficients validate."""
-        RecursiveFilter(edge_width=1)._validate_smoothing_coefficients(
-            self.cube[0, :], self.smoothing_coefficients_cube_x
+        _ = RecursiveFilter(edge_width=1)._validate_and_pad_coefficients(
+            self.cube, self.smoothing_coefficients
+        )
```

Comment:
Since this only tests that no value error is raised, I think this is now a duplicate (as it's covered by the test below).

Reply:
Agreed. Removed.